spec/lib/wifi_user/use_cases/email_signup_spec.rb

```diff
@@ -117,10 +119,25 @@
 
       context "given an email address with a non-gov domain, but email is sponsored" do
         let(:allowlist_checker) { double(execute: { success: false }) }
-        let(:check_user_is_sponsee) { double(execute: true) }
-        let(:created_contact) { "ryan@example.com" }
-        let(:username) { "irrelevant" }
-        let(:password) { "irrelephant" }
+        let(:sponsee_is_user_checker) { double(execute: true) }
```

Comment:
I don't think any of these, apart from maybe created_contact require `let!` do they? Better use plain `let` I think